recipes/sentry-breakpad/all/conandata.yml

```diff
@@ -0,0 +1,17 @@
+sources:
+  "0.4.9":
+    url: "https://github.com/getsentry/sentry-native/releases/download/0.4.9/sentry-native.zip"
+    sha256: "559344bad846b7868c182b22df0cd9869c31ebf46a222ac7a6588aab0211af7d"
```

Comment:
do you need all these minor versions?

Reply:
They're the same versions the `sentry-native` recipe provides.

Reply:
We can create them once, but they might be removed from the repository in the future (like patch versions from `cmake`, `meson`,...). IMO It can be good to have them available in the remote for at least one revision.